src/blue_http.ml

```diff
@@ -1,25 +1,31 @@
 open Core
 open Async
+module Client = Client
 
 let set_default_max_redirects = Redirect.set_default_max_redirects
+let default_client = lazy (Client.create ())
 
-let request_stream ?max_redirects ?interrupt ?headers ?chunked ?body meth uri =
+let request_stream ?max_redirects ?interrupt ?headers ?chunked ?body ?client meth uri =
+  let client =
+    match client with
+    | Some client -> client
+    | None -> Lazy.force default_client
```

Comment:
Or maybe the default should be creating a new client and then closing it at the end?